Metadata/Driver/AnnotationDriver.php

```diff
@@ -175,7 +180,7 @@ public function loadMetadataForClass(\ReflectionClass $class)
                         $params[] = $this->convertReferenceValue($paramName, $annot->params[$paramName]);
                     }
 
-                    if (!$params) {
+                    if (!$params && !$annot instanceof Call) {
                         continue;
```

Comment:
This bundle is mostly about injection, I do not see why you would call something without an argument unless it's some sort of initializer. Is that the case here? If so, this is what the ``@AfterSetup`` method annotation did.

Is there a use-case for having more than one of those methods per-class?